pkg/common/parameters.go

```diff
@@ -22,11 +22,15 @@ import (
 )
 
 const (
+	// Parameters for StorageClass
 	ParameterKeyType                 = "type"
 	ParameterKeyReplicationType      = "replication-type"
 	ParameterKeyDiskEncryptionKmsKey = "disk-encryption-kms-key"
 	ParameterKeyLabels               = "labels"
 
+	// Parameters for VolumeSnapshotClass
+	ParameterStorageLocations = "storage-locations"
```

Comment:
For consistency this could be `ParameterKeyStorageLocations` but it's just a nit

Reply:
Done.